Code/GraphMol/MolOps.cpp

```diff
@@ -967,6 +1008,128 @@ unsigned getNumAtomsWithDistinctProperty(const ROMol &mol, std::string prop) {
   return numPropAtoms;
 }
 
+namespace {
+bool checkNeighbors(const Bond *bond, const Atom *atom) {
+  PRECONDITION(bond, "no bond");
+  PRECONDITION(atom, "no atom");
+  std::vector<int> nbrRanks;
+
+  for (auto nbrBond : bond->getOwningMol().atomBonds(atom)) {
+    if (nbrBond == bond) {
+      continue;  // a bond is NOT its own neighbor
+    }
+
+    if (nbrBond->getBondType() == Bond::SINGLE) {
+      // if a neighbor has a wedge or hash bond, do NOT mark it as double
+      // crossed
+      if (nbrBond->getBondDir() == Bond::ENDUPRIGHT ||
+          nbrBond->getBondDir() == Bond::ENDDOWNRIGHT) {
+        return false;
+      }
+
+      // if a neighbor has a wiggle bond, do NOT mark it as crossed (although it
+      // is unknown
+      if (nbrBond->getBondDir() == Bond::BondDir::UNKNOWN &&
+          nbrBond->getBeginAtom() == atom) {
+        return false;
+      }
+
+      // if two neighbors has the same CIP ranking, this is not stereo
+
+      const auto otherAtom = nbrBond->getOtherAtom(atom);
+      int rank;
+      if (RDKit::Chirality::getUseLegacyStereoPerception()) {
+        if (!otherAtom->getPropIfPresent(common_properties::_CIPRank, rank)) {
+          rank = (-1);
```

Comment:
OK changed and in a couple of other places too.  This is an old habit from "C" where "=-" used to be an operator, so that i=-1 was different than i = -1.  LIke ducking under a tree branch that you hit you head on, even twenty years after the branch was cut down.

Reply:
AndI have pushed the changes from the latest PR review